### PR TITLE
Dev -> Main October 2024

### PR DESCRIPTION
**Pull request by maxachis**
## Fixes:

- https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/162
- https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/250
- https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/451
- https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/450
- https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/417
- https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/253
- https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/433
- https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/434
- https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/442
- https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/443
- https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/445
- https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/446
- https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/457
- https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/462
- https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/256
- https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/306
- https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/390
- https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/406
- https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/419
- https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/424
- https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/428
- https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/436

## Contains:

- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/83
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/84
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/85
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/86
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/87
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/88
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/89
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/90
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/91
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/92
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/93
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/94
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/95
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/96
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/97
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/98
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/99
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/100
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/101
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/102
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/103
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/104
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/105
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/106
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/107
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/108
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/109
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/110
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/111
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/112
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/113
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/114
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/115
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/116
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/117
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/118
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/119
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/120
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/121
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/122
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/123
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/124
- https://github.com/Police-Data-Accessibility-Project/data-sources-app-v2/pull/126

### CODE REVIEW

**Review by josh-chamberlain — COMMENTED** [review]
- [x] Using Chrome, open [developer tools](https://developer.chrome.com/docs/devtools/overview) and navigate to the "Console" tab to enable visibility of client-side errors.
- [x] Navigate to the domain and confirm the home page loads as expected without error
- [x] Perform a search in the database using a common keyword such as "Prison" or "Pittsburgh". See that results are returned without error.
- [x] For a given result, click on "More Details" and see that results are loaded.
- [x] Click on the nav links and confirm they load the expected page without error
- [x] Perform the above in both the desktop version and a mobile device

**Review by josh-chamberlain — APPROVED** [review]
I did the janitorial work, formatted with Black, and this looks good to go! I'll let you merge when you're next online.

**Comment by maxachis** [comment]
> I did the janitorial work, formatted with Black, and this looks good to go! I'll let you merge when you're next online.

@josh-chamberlain DONE 🔥

**Pull request merged**
